### PR TITLE
Adding Coordflip for one variable

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

File: instat/static/InstatObject/R/data_object_R6.R [hunk 1]
```diff
@@ -1635,6 +1638,9 @@ data_object$set("public", "graph_one_variable", function(columns, numeric = "geo
       else {
         g <- ggplot(data = curr_data, mapping = aes_(x = as.name(column))) + ylab("")
       }
+      if (coord_flip){
+        g <- g + coord_flip()
+      }
```
Comment (dannyparsons) [inline instat/static/InstatObject/R/data_object_R6.R]: This looks great. Just need a space before each `{` for consistent layout.

Comment (maxwellfundi) [inline instat/static/InstatObject/R/data_object_R6.R]: Okay. Am on it.

File: instat/static/InstatObject/R/instat_object_R6.R [hunk 2]
```diff
@@ -972,8 +972,8 @@ instat_object$set("public","set_column_colours_by_metadata", function(data_name,
 }
 )
 
-instat_object$set("public","graph_one_variable", function(data_name, columns, numeric = "geom_boxplot", categorical = "geom_bar", character = "geom_bar", output = "facets", free_scale_axis = FALSE, ncol = NULL, ...) {
-  self$get_data_objects(data_name)$graph_one_variable(columns = columns, numeric = numeric, categorical = categorical, output = output, free_scale_axis = free_scale_axis, ncol = ncol, ... = ...)
+instat_object$set("public","graph_one_variable", function(data_name, columns, numeric = "geom_boxplot", categorical = "geom_bar", character = "geom_bar", output = "facets", free_scale_axis = FALSE, ncol = NULL, coord_flip  = FALSE,...) {
```
Comment (dannyparsons) [inline instat/static/InstatObject/R/instat_object_R6.R]: Last thing, need a space before `...`

**Comment by rdstern** [comment]
@maxwellfundi Seems to work fine.  Great.  I tried with the ggplot diamonds data set.

Asking for a pie charts gives an error.  This isn't related to the coord-flip.  I think it may be looking for a geom to do the pie chart, when it is a polar coordinates on the bar chart that it needs instead.  I thought this was working before, but may have been mistaken?

**Comment by maxwellfundi** [comment]
I will have a look to fix that @rdstern

**Pull request merged**